src/components/Web3Status/ConnectWalletPopover.tsx

```diff
@@ -128,7 +128,8 @@ export const ConnectWalletPopover = ({ setModal, tryActivation, children }: Conn
                 closeModals()
                 option.connector !== connector && !option.href && tryActivation(option.connector)
               }}
-              icon={require('../../assets/images/' + option.iconName)}
+              // eslint-disable-next-line
+              icon={require('../../assets/images/' + option.iconName).default}
```

Comment:
I would do the same here

Reply:
Kind of an overkill with `require`, and masks the explicit ESlint rule.